genetic_algorithm/one_max.py

```diff
@@ -0,0 +1,233 @@
+from collections.abc import Callable
+
+from numpy.random import rand, randint
+
+"""
+This program performs genetic evolution on a list of bits.
+Its goal is to transform every bit to a 1, using mutations and crossovers.
+https://en.wikipedia.org/wiki/Genetic_algorithm
+"""
+
+
+def goal(individual: list[int]) -> int:
+    """
+    Calculate the fitness score for an individual in the genetic evolution algorithm.
+
+    The goal of the algorithm is to transform every bit to a 1, and the fitness
+    score is the negation of the sum of the bits.
+
+    Parameters:
+    - individual (list[int]): A list representing an individual in the population.
```

Comment:
Please do not repeat the type hints already stated in the function signature and verified by mypy because readers will be confused if one changes and the other does not.  Repeat below.
```suggestion
    - individual: A list representing an individual in the population.
```